contrib/cloud.google.com/go/pubsub.v1/pubsub.go

```diff
@@ -80,6 +99,16 @@ func WrapReceiveHandler(s *pubsub.Subscription, f func(context.Context, *pubsub.
 			tracer.Tag("message_id", msg.ID),
 			tracer.Tag("publish_time", msg.PublishTime.String()),
 			tracer.ChildOf(parentSpanCtx),
+		}
+
```

Comment:
Please remove this empty line.

Reply:
Is there a style guide available that I can reference?

Reply:
Unfortunately not, sorry about that. I have to play human linter. I don't think it's easy to create a style guide that specifically references how to use empty lines. We're mostly trying to follow the same style as the standard library code. You'll notice that it's not very common to use a lot of spacing in Go, especially around curly braces which already create spacing on their own (being just one character on a line e.g. `}`)